src/pubsub/pubsub_form_utils.erl

```diff
@@ -203,7 +203,7 @@ convert_value_from_binaries([Bin], boolean) ->
 convert_value_from_binaries([Bin], integer) ->
     binary_to_integer(Bin);
 convert_value_from_binaries([Bin], datetime) ->
-    jlib:datetime_binary_to_timestamp(Bin);
+    usec:to_now(calendar:rfc3339_to_system_time(binary_to_list(Bin), [{unit, microsecond}]));
```

Comment:
I can see that other replacements of `jlib:datetime_binary_to_timestamp` were surrounded by `try ... catch` or `case catch`. We don't have it here, are we sure at this point that the conversion will always succeed?

Reply:
This function is already inside a `try ... catch` in line 186.